packages/web-app-files/src/helpers/resource/move.ts

```diff
@@ -203,14 +255,25 @@ export const move = async (
       }
       if (resolveStrategy === ResolveStrategy.KEEP_BOTH) {
         targetName = $gettextInterpolate($gettext('%{name} copy'), { name: resource.name }, true)
+        resource.name = targetName
       }
     }
+    resource.path = join(targetFolder.path, resource.name)
```

Comment:
join from 'path' is a node module, please take care that it is polyfilled

Reply:
it is imported via `import { join } from 'path'` ^^ or do you mean something else?

Reply:
nah its fine, i had a look. Path is a node module and is not available in browsers context, thats the reason why we have to polyfill it.... anyway i had a look and were already polyfilled it :)